.github/workflows/ci.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v3
         with:
-          go-version: 1.18.5
```

Comment:
revert if this is not required

Reply:
after updating go.mod, need to upgrade to go.1.19
```
# go.uber.org/multierr
Error: ../../../go/pkg/mod/go.uber.org/multierr@v1.11.0/error.go:209:20: undefined: atomic.Bool
note: module requires Go 1.19
# go.opentelemetry.io/otel/sdk/trace
Error: ../../../go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.16.0/trace/provider.go:78:24: undefined: atomic.Pointer
Error: ../../../go/pkg/mod/go.opentelemetry.io/otel/sdk@v1.16.0/trace/provider.go:80:20: undefined: atomic.Bool
note: module requires Go 1.19
Error: Process completed with exit code 2.
```